<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1A1F33;</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>621</width>
       <height>276</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="6">
       <widget class="QPushButton" name="pushButton_6">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #353A4E;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_35">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>Credit</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_22">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>acos</string>
        </property>
       </widget>
      </item>
      <item row="3" column="7">
       <widget class="QPushButton" name="pushButton_15">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="pushButton_20">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QPushButton" name="pushButton_12">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #4BC3FE;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_26">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="pushButton_10">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #353A4E;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButton_23">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>atan</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QPushButton" name="pushButton_3">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #353A4E;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_30">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>(</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QLabel" name="label_xMin">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>xMin</string>
        </property>
       </widget>
      </item>
      <item row="4" column="8">
       <widget class="QLabel" name="label_yMax">
        <property name="styleSheet">
         <string notr="true">QLabel {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>yMax</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_28">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>sqrt</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QPushButton" name="pushButton_8">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #353A4E;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="pushButton_4">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #353A4E;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_21">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>asin</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_33">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QPushButton" name="pushButton_14">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButton_18">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>cos</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true">background-color: #353A4E;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="pushButton_7">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #353A4E;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pushButton_19">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>tan</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_34">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>Deposit</string>
        </property>
       </widget>
      </item>
      <item row="4" column="7">
       <widget class="QPushButton" name="pushButton_16">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pushButton_29">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>log</string>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QLabel" name="label_xMax">
        <property name="styleSheet">
         <string notr="true">QLabel {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>xMax</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QPushButton" name="pushButton_11">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #353A4E;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="styleSheet">
         <string notr="true">border: 2px solid #353A4E;
border-radius: 7px;
max-width: 60px;
padding: 7px 0 7px;
margin: 0 1px 0;
color: white;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_27">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>ln</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="pushButton_5">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #353A4E;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_17">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>sin</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="pushButton_2">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #353A4E;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QPushButton" name="pushButton_13">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="3" column="6">
       <widget class="QPushButton" name="pushButton_9">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #353A4E;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="8">
       <widget class="QLabel" name="label_yMin">
        <property name="styleSheet">
         <string notr="true">QLabel {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>yMin</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QLineEdit" name="xMin">
        <property name="styleSheet">
         <string notr="true">border: 2px solid #353A4E;
border-radius: 7px;
max-width: 40px;
padding: 7px 0 7px;
margin: 0 1px 0;
color: white;</string>
        </property>
        <property name="text">
         <string>-10</string>
        </property>
       </widget>
      </item>
      <item row="2" column="9">
       <widget class="QLineEdit" name="xMax">
        <property name="styleSheet">
         <string notr="true">border: 2px solid #353A4E;
border-radius: 7px;
max-width: 40px;
padding: 7px 0 7px;
margin: 0 1px 0;
color: white;</string>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item row="3" column="9">
       <widget class="QLineEdit" name="yMin">
        <property name="styleSheet">
         <string notr="true">border: 2px solid #353A4E;
border-radius: 7px;
max-width: 40px;
padding: 7px 0 7px;
margin: 0 1px 0;
color: white;</string>
        </property>
        <property name="text">
         <string>-10</string>
        </property>
       </widget>
      </item>
      <item row="4" column="9">
       <widget class="QLineEdit" name="yMax">
        <property name="styleSheet">
         <string notr="true">border: 2px solid #353A4E;
border-radius: 7px;
max-width: 40px;
padding: 7px 0 7px;
margin: 0 1px 0;
color: white;</string>
        </property>
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QPushButton" name="pushButton_24">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>^</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QPushButton" name="pushButton_25">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>⌫</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QPushButton" name="pushButton_31">
        <property name="styleSheet">
         <string notr="true">QPushButton {
background-color: #2C3145;
color: white;
font-family: 'JetBrains Mono';
padding: 15px 0 15px;
border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>Graph</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="7">
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">border: 2px solid #353A4E;
border-radius: 7px;
padding: 7px 0 7px;
margin: 0 5px 0;
color: white;
font-family: Jetbrains Mono;
</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>371</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
